socialhome/streams/app/components/AuthorBar.vue

```diff
@@ -61,31 +58,59 @@ export default Vue.component("author-bar", {
         isUserAuthentificated: {type: Boolean, required: true},
     },
     data() {
-        return {showProfileBox: false}
+        return {
+            showProfileBox: false,
+            _isUserFollowingAuthor: this.isUserFollowingAuthor,
+        }
     },
     computed: {
+        isUserRemote(){
+            return !this.isUserLocal
+        },
         showFollowBtn() {
-            return this.isUserAuthentificated
-                    && !this.isUserLocal
-                    && !this.isUserFollowingAuthor
+            return this.canFollow && !this.$data._isUserFollowingAuthor
```

Comment:
Is `this.$data` the correct way to do this? Tried looking and can't find much. So trusting you on this 👍 Just looks weird and wondering if it should be used directly, since isn't it calculated by `data()` ?